kcidb/cache/__init__.py

```diff
@@ -52,11 +52,13 @@ def store(self, url):
 
         # Cache every 4096th URL only for the trial period
         if not object_name.endswith("000"):
+            LOGGER.debug("Skipping URL '%s', hash not matched.", url)
             return
 
         blob = self.client.bucket(self.bucket_name).blob(object_name)
 
         if blob.exists():
+            LOGGER.debug("URL '%s' already exists in cache.", url)
```

Comment:
Please add `, not caching` on the end, so it's clear what's the outcome. Here, and in the following two messages.

Reply:
Okay